### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by tristann3**

### CODE REVIEW

**Review by starlightromero — CHANGES_REQUESTED** [review]
It looks like you are still using handlebars. Is there a reason for this?

File: tests/appTest.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const app = require('../app.js');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const cleaner = require('../utils');
+
+const { assert } = chai;
+const fetch = require('node-fetch');
```
Comment (Dino-Martinez) [inline tests/appTest.js]: We don't need node-fetch in this file

File: package.json [hunk 2]
```diff
@@ -21,14 +21,19 @@
   "license": "MIT",
   "dependencies": {
     "axios": "^0.21.1",
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "express-handlebars": "^5.2.0",
+    "mongoose": "^5.13.0",
```
Comment (Dino-Martinez) [inline package.json]: We don't need mongoose thus far

**Review by Dino-Martinez — CHANGES_REQUESTED** [review]
uninstall mongoose/handlebars and remove a couple unnecessary lines

File: package.json (hunk 2 above)
Comment (tristann3) [reply package.json]: Removed dependency

File: tests/appTest.js (hunk 1 above)
Comment (tristann3) [reply tests/appTest.js]: Removed import

**Comment by tristann3** [comment]
> It looks like you are still using handlebars. Is there a reason for this?

No, was old code, removed it

**Review by Dino-Martinez — APPROVED** [review]

**Review by starlightromero — APPROVED** [review]

**Pull request merged**